ui/src/treeDataProvider/AzureParentNode.ts

```diff
@@ -126,7 +126,7 @@ export class AzureParentNode<T extends IAzureParentTreeItem = IAzureParentTreeIt
         if (this.treeItem.createChild && this.treeItem.childTypeLabel) {
             picks.unshift(new PickWithData<GetNodeFunction>(
                 async (): Promise<AzureNode> => await this.createChild(),
-                localize('azFunc.NodePickerCreateNew', '$(plus) Create New {0}', this.treeItem.childTypeLabel)
+                localize('azureextensionui.NodePickerCreateNew', '$(plus) Create New {0}', this.treeItem.childTypeLabel)
```

Comment:
We've actually been getting rid of these prefixes (azFunc, etc.) for localization because it shouldn't be necessary (I started the pattern - I know it's my fault lol). The id only needs to be unique on a per-file basis. You can see [this example](https://github.com/Microsoft/vscode-node-debug/tree/master/i18n/esn/out/node/extension) for what the localized file structure eventually looks like